.github/workflows/regression-public.yml

```diff
@@ -0,0 +1,74 @@
+name: Run regression tests
+
+on:
+  workflow_call:
+    inputs:
+      environment:
+        description: "Notification-api deployment environment to test (dev|perf|staging)"
+        default: "perf"
+        required: true
+        type: string
+      qa-image-branch:
+        description: "Will run latest SHA from branch"
+        default: "master"
+        required: true
+        type: string
+
+jobs:
+  run-QA-suite:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.qa-image-branch }}
+
+      - name: Configure VAEC AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v2.2.0
+        with:
+          aws-access-key-id: ${{ secrets.VAEC_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.VAEC_AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-gov-west-1
+          role-to-assume: ${{ secrets.VAEC_DEPLOY_ROLE }}
+          role-skip-session-tagging: true
+          role-duration-seconds: 900
+
+      - name: Login to VAEC ECR
+        id: login-ecr-vaec
+        uses: aws-actions/amazon-ecr-login@v1
+        with:
+          mask-password: 'true'
+
+      - name: Install jq
+        run: |
+          sudo apt install jq -y
+
+      - name: QA regression test - public endpoints
+        env:
+          ECR_REGISTRY: ${{ steps.login-ecr-vaec.outputs.registry }}
+        run: |
+          # Add user qa-container with UID 1051 and home directory at /home/qa-container
+          sudo useradd -u 1051 -m -d /home/qa-container -s /bin/bash qa-container
+
+          ## Set the working directory to /home/qa-container
+          cd /home/qa-container
+
+          #aws ssm get-parameter --name /notification-api-qa/env --with-decryption | sudo tee .env > /dev/null
```

Comment:
Is this commented-out code left in here intentionally?

Reply:
good catch.  I've updated the comments on the file, please check again